Gifski/TrimmingAVPlayerView.swift

```diff
@@ -0,0 +1,47 @@
+import AVKit
+
+final class TrimmingAVPlayerView: AVPlayerView {
+	private var timeRangeObserver: NSKeyValueObservation?
+
+	func observeTrimmedTimeRange(_ updateClosure: @escaping (ClosedRange<Double>) -> Void) {
+		timeRangeObserver = player?.currentItem?.observe(\.duration, options: [.new]) { item, _ in
+			let startTime = item.reversePlaybackEndTime.seconds
+			let endTime = item.forwardPlaybackEndTime.seconds
+			if !startTime.isNaN && !endTime.isNaN {
+				updateClosure(startTime...endTime)
+			}
+		}
+	}
+
+	func hideTrimButtons() {
```

Comment:
I would make this method private and call it inside `func viewDidMoveToSuperview() {}` in this class. Making it more self-contained is nice.

Reply:
For some reason `viewDidMoveToSuperview` is not called - maybe due to the view being created in xib?

Reply:
`func awakeFromNib()` then?

Reply:
It's too early to find the buttons - same thing when I tried just after `playerView.beginTrimming`

maybe if we transformed `TrimmingAVPlayerView` into a VC it would work?

Reply:
Yeah, most often doing a VC is a better option.

Reply:
It worked! Pushed the update.